app/models/metric/capture.rb

```diff
@@ -234,4 +238,5 @@ def self.minutes_ago(value)
       value.to_i_with_method.seconds.ago.utc
     end
   end
+  private_class_method :minutes_ago
```

Comment:
@Fryguy, I left this one the way it was using private_class_method and updated the other PRs #11031 and #11032